src/cpu/reorder/cpu_reorder_pd.hpp

```diff
@@ -69,7 +69,8 @@ struct cpu_reorder_pd_t : public reorder_pd_t {
             *D_mask = utils::array_product(
                     input_d.dims() + ndims_start, ndims_mask);
         assert(*D_mask >= 1);
```

Comment:
Thanks for the patch. Please put this `assert` under `if` above, otherwise this could explode in debug mode as well.

Reply:
Done